README.md

```diff
@@ -67,6 +67,13 @@ mysql> grant all privileges on *.* to 'hyfuser'@'localhost';
 mysql> create database userdb;
 ```
 
+**Step 3: Installing MySQL drivers for node**
+
+We want to use MySQL with JavaScript and to this end, we use `mysqljs` driver.
+* Use `node install mysql` command to install them.
```

Comment:
It should be `npm install mysql`. I'd also consider asking students to install it globally, by adding the `-g` to it:

`npm install -g mysql`

Reply:
Thanks for spotting this. I have added the command